python/graphstorm/gsf.py

```diff
@@ -338,6 +340,46 @@ def create_builtin_reconstruct_nfeat_decoder(g, decoder_input_dim, config, train
     loss_func = RegressionLossFunc()
     return decoder, loss_func
 
+# pylint: disable=unused-argument
+def create_builtin_reconstruct_efeat_decoder(g, decoder_input_dim, config, train_task):
+    """ create builtin edge feature reconstruction decoder
+        according to task config
+
+    Parameters
+    ----------
+    g: DGLGraph
+        The graph data.
+        Note(xiang): Make it consistent with create_builtin_edge_decoder.
```

Comment:
In the #370 line, the ``g`` is used. So why here say, it is reserved for future?